rest/data/patch.go

```diff
@@ -342,7 +346,7 @@ func (hp *MockPatchConnector) FindPatchesByProject(projectId string, ts time.Tim
 	}
 	for i := len(hp.CachedPatches) - 1; i >= 0; i-- {
 		p := hp.CachedPatches[i]
-		if *p.ProjectId == projectId && !p.CreateTime.After(ts) {
+		if (*p.ProjectId == projectId || *p.ProjectIdentifier == projectId) && !p.CreateTime.After(ts) {
```

Comment:
instead can we can just check if *p.ProjectId == id

Reply:
done